docs/devGuide/development/settingUp.md

```diff
@@ -34,7 +34,7 @@ This page explains how to set up your development environment to start contribut
 
 <box type="tip" seamless>
 
-We recommend the **WebStorm IDE** for working with MarkBind code.
+We recommend the **WebStorm IDE** or **VS Code** for working with MarkBind code.
```

Comment:
Small nit: should we include links to both IDEs so its easier for users?

Reply:
It's quite silly but when I thought about it I wasn't sure if I liked the look of it being being bolded and a link 😂 I was also thinking about if the user would expect it to link to something within the MarkBind website and be taken by surprise by a link that goes out

Reply:
Fair enough